### PR TITLE
Fix of section 4

### PR DESCRIPTION
**Pull request by gsusag00**

### CODE REVIEW

**Review by uo276077 — CHANGES_REQUESTED** [review]
Suggested a few grammar changes

File: docs/04_solution_strategy.adoc [hunk 1]
```diff
@@ -5,8 +5,11 @@
 
 A summary of the technical decisions that we have taken so far:
 
-    * **React**: A framework to build the user interface of the application.
     * **MongoDB**: The DMBS of our choice.
+    * **Express**: Express is a framework to handle the backend of our application.
+    * **Redux**: Though it hasn't been implemented yet, we are considering it.
```
[the remark this answers is not in the document]
Comment (gsusag00) [reply docs/04_solution_strategy.adoc:10]: Noted

File: docs/04_solution_strategy.adoc [hunk 2]
```diff
@@ -5,8 +5,11 @@
 
 A summary of the technical decisions that we have taken so far:
 
-    * **React**: A framework to build the user interface of the application.
     * **MongoDB**: The DMBS of our choice.
```
[the remark this answers is not in the document]
Comment (gsusag00) [reply docs/04_solution_strategy.adoc:8]: Noted

**Review by uo276077 — APPROVED** [review]
With correct changes, approved file

**Pull request merged**